File: 前端项目/ModelTest/Assets/MvRAssets/CS_Assets/CS_Scripts/Spin.cs
using UnityEngine;

namespace MvR
{
	/// <summary>
	/// A simple script to rotate an object at a certain speed.
	/// </summary>
	public class Spin : MonoBehaviour
	{
		public float rotateSpeed = 2;
	
		/// <summary>
		/// Update is called every frame, if the MonoBehaviour is enabled.
		/// </summary>
		public void Update()
		{
			transform.eulerAngles = new Vector3(transform.eulerAngles.x, transform.eulerAngles.y + (rotateSpeed * Time.deltaTime));
		}
	}
}

File: 前端项目/ModelTest/Assets/MvRAssets/JS_Assets/JS_Scripts/Spin.js
//A simple script to rotate an object at a certain speed

#pragma strict

private var thisTransform:Transform;
var rotateSpeed:float = 2;

function Start() 
{
	thisTransform = this.transform;
}

function Update() 
{
	thisTransform.eulerAngles.y += rotateSpeed * Time.deltaTime;
}